torch/csrc/api/src/mps.cpp

```diff
@@ -25,9 +21,22 @@ void manual_seed(uint64_t seed) {
 }
 
 void synchronize() {
-  TORCH_CHECK(is_available(), "No MPS devices are available");
   at::detail::getMPSHooks().deviceSynchronize();
 }
 
+void commit() {
+  at::detail::getMPSHooks().commitStream();
+}
+
+MTLCommandBuffer_t get_command_buffer() {
+  return static_cast<MTLCommandBuffer_t>(
+      at::detail::getMPSHooks().getCommandBuffer());
```

Comment:
Static cast is redundant here, isn't it?
```suggestion
  return at::detail::getMPSHooks().getCommandBuffer();
```

Reply:
The `getMPSHooks().getCommandBuffer()` returns a `void*` (since it comes from MPSHooksInterface.cpp which is CPP). So we need to cast it to the correct pointer type at this stage before returning it to user.

Reply:
Updated the PR and removed the explicit cast.